src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,41 @@
-export const Tabs = () => {};
+import React from 'react';
+
+type Tab = {
+  id: string;
+  title: string;
+  content: string;
+};
+
+interface Props {
+  tabs: Tab[];
+  selectedTab: Tab;
```

Comment:
let's pass `selectedTabId` instead, and then will try to find one among all tabs